zerver/tests/test_subs.py

```diff
@@ -2676,7 +2759,7 @@ def test_realm_admin_remove_others_from_subbed_private_stream(self) -> None:
         are on.
         """
         result = self.attempt_unsubscribe_of_principal(
-            query_count=17,
+            query_count=18,
```

Comment:
This query count increase happens only when unsubscribing user from a private stream because we send `stream/delete` event which includes the complete stream information including `can_remove_subscribers_group` value. 

We have discussed about just sending stream IDs with the delete event since clients only use IDs.

Reply:
OK -- can you make sure we have an issue tracking that follow-up, maybe with the "performance" label? It doesn't seem urgent, but would be good to not forget this understanding of an easy optimization we can do.

Reply:
Opened #32369.